docs/integrations/microsoft-azure/azure-storage.md

```diff
@@ -73,6 +163,7 @@ In this section, you will configure a pipeline for shipping metrics from Azure M
 
 ### Configure logs collection
 
+#### Diagnostic logs
 In this section, you will configure a pipeline for shipping diagnostic logs from Azure Monitor to an Event Hub.
 
 1. To set up the Azure Event Hubs source in Sumo Logic, refer to [Azure Event Hubs Source for Logs](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/).
```

Comment:
Tag the location field in the source. @sachin-sumologic  please add screenshot here